clang/lib/Tooling/Inclusions/Stdlib/StdSymbolMap.inc

```diff
@@ -1241,8 +1254,7 @@ SYMBOL(fgetwc, None, <wchar.h>)
 SYMBOL(fgetws, std::, <cwchar>)
 SYMBOL(fgetws, None, <cwchar>)
 SYMBOL(fgetws, None, <wchar.h>)
-SYMBOL(filebuf, std::, <streambuf>)
-SYMBOL(filebuf, std::, <iostream>)
+SYMBOL(filebuf, std::, <fstream>)
```

Comment:
Previous versions of cppreference mentioned incorrect headers for `filebuf` and `wfilebuf`.
`<fstream>` is the correct one: https://eel.is/c++draft/fstream.syn#lib:filebuf